Examples/src/main/java/com/mapbox/vision/examples/ArActivity.java

```diff
@@ -344,4 +341,46 @@ public void userOffRoute(Location location) {
     private List<Point> buildStepPointsFromGeometry(String geometry) {
         return PolylineUtils.decode(geometry, Constants.PRECISION_6);
     }
+
+    private ManeuverType mapToManeuverType(@Nullable String maneuver) {
+        if (maneuver == null) {
+            return ManeuverType.None;
+        }
+        switch (maneuver) {
+            case "turn":
+                return ManeuverType.Turn;
+            case "depart":
+                return ManeuverType.Depart;
+            case "arrive":
+                return ManeuverType.Arrive;
+            case "merge":
+                return ManeuverType.Merge;
+            case "on ramp":
+                return ManeuverType.OnRamp;
+            case "off ramp":
+                return ManeuverType.OffRamp;
+            case "fork":
+                return ManeuverType.Fork;
+            case "roundabout":
+                return ManeuverType.Roundabout;
+            case "exit roundabout":
+                return ManeuverType.RoundaboutExit;
+            case "end of road":
+                return ManeuverType.EndOfRoad;
+            case "new name":
+                return ManeuverType.NewName;
+            case "continue":
+                return ManeuverType.Continue;
+            case "rotary":
+                return ManeuverType.Rotary;
+            case "roundabout turn":
+                return ManeuverType.RoundaboutTurn;
+            case "notification":
+                return ManeuverType.Notification;
+            case "exit rotary":
+                return ManeuverType.RotaryExit;
+            default:
+                return ManeuverType.None;
```

Comment:
Collapse to single line? Up to you

Reply:
Naahh, tried, did not like it collapsed.